docs/joss_paper/paper.md

```diff
@@ -50,72 +62,72 @@ affiliations:
+DeepReg provides command-line tool options that enable a set of basic and advanced functionalities for model training, prediction and image warping. The yaml files and command-line arguments are used for configuring the various options with DeepReg. This design choice has been motivated by both research and education purposes originally set out in the DeepReg project, which aims for open and easy access to the benchmarking implementations of standard deep-learning-based image registration algorithms. These implementations, together with their documentation, tutorials and demos, simplify workflows for prototyping and developing novel methodology, rapid utilising latest development in this field and accessing quality research advances. A MICCAI Educational Challenge has utilised the DeepReg code and demos to explore the link between classical algorithms and deep-learning-based methods [@brown2020introduction], while a recently published research paper investigated temporal changes for prostate cancer patients under active surveillance programme, by using longitudinal image registration adapted from the DeepReg code [@yang2020longitudinal].
 
 # Algorithms
-In this paper, we first summarise several standard pairwise image registration network training strategies, in which the network aims to align a pair of moving- and fixed images such that the moving image can be warped or transformed into the fixed image coordinates. The resulting warped moving image is also denoted as predicted fixed image (or pred_fixed_image) in the `DeepReg` source code. The methodologies adopted in these algorithms are building blocks of many other registration tasks, such as group-wise registration.
+In this paper, we first summarise several standard pairwise image registration network training strategies, in which the network aims to align a pair of moving- and fixed images such that the moving image can be warped or transformed into the fixed image coordinates. The resulting warped moving image is also denoted as predicted fixed image or "*pred_fixed_image*" in the DeepReg source code. The methodologies adopted in these algorithms are building blocks of many other registration tasks, such as group-wise registration and morphological template construction [@dalca2019learning; @siebert2020deep; @luo2020mvmm].
```

Comment:
Suggestion: only in weak-supervised learning is mentioned the regularization loss. We may instead want to included here as a general:
It would read as follows (line 2):  ... or "*pred_fixed_image*" in the DeepReg source code. All different strategies may include regularisation on the smoothness of the predicted displacement field. The methodologies adopted ...